[DOCS] Use same certificate on every node (#72900)
elastic/elasticsearch: x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

. Open the `ES_PATH_CONF/elasticsearch.yml` file and make the following
changes:

   a. Add the <<cluster-name,`cluster-name`>> setting and enter a name for your cluster:
+
[source,yaml]
----
cluster.name: my-cluster
----

b. Add the <<node-name,`node.name`>> setting and enter a name for the node.
The node name defaults to the hostname of the machine when {es} starts. 
+
[source,yaml]
----
node.name: node-1
----

c. Add the following settings to enable internode communication and provide
access to the node's certificate.
+

> Because you are using the `elastic-certificates.p12` file on every node in

Reviewer: ```suggestion
Because you are using the same `elastic-certificates.p12` file on every node in
```

your cluster, set the verification mode to `certificate`:
+
[source,yaml]
----
xpack.security.transport.ssl.enabled: true
xpack.security.transport.ssl.verification_mode: certificate <1>
xpack.security.transport.ssl.client_authentication: required
xpack.security.transport.ssl.keystore.path: elastic-certificates.p12
xpack.security.transport.ssl.truststore.path: elastic-certificates.p12
----
<1> If you want to use hostname verification, set the verification mode to

> `full`. You'll need to generate a different certificate for each host that

Reviewer: Strictly, this isn't true, but I don't know what we should actually say...

One of the recurring challenges with these docs is that most readers want something prescriptive: _Follow these steps and it will all work correctly for you_.

However, there is the occasional reader who has their own existing plan/process and they want to understand whether it will work (and is supported). Overly prescriptive language hinders them because they don't want to do the thing we're telling people to do, they just want to know what options are supported.

For 99+% of people that want hostname verification, a certificate per node is the right option. However, technically it's possible to generate a single certificate with multiple hostnames in it and use that across the cluster, or use a wildcard certificate. I really don't want us to explain that here - it will turn a short explanatory note into a whole page doc and reduce readability. However, I predict someone, at some point in the future, will ask _Why do you say I need to have a certificate per host when a single certificate with multiple IP addresses works fine_. 

Maybe we can get away with:
```suggestion
`full`. You should generate a different certificate for each host that
```
?
